ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -575,6 +580,26 @@ import { DraggableListBehavior } from '../nuxeo-draggable-list-behavior.js';
       });
     }
 
+    _computeActionsStyle() {
+      afterNextRender(this, () => this._alignHeaderRow());
+      if (this.editable && this.orderable) {
+        return `flex: 0 0 172px;`;
```

Comment:
Did this passed on code style? We shouldn't be using different delimiters for this kind of situation.

Reply:
It passed on code style. I've changed it already.